Makefile.am

```diff
@@ -2,25 +2,20 @@ AM_CFLAGS = -I$(srcdir)/include
 ACLOCAL_AMFLAGS = -I config
 
 bin_PROGRAMS = \
-################################################################################
-# Multi-node bins
-################################################################################
 	performance/multi-node/rdm_bw \
 	performance/multi-node/rdm_latency \
 	performance/multi-node/rdm_mbw_mr \
 	performance/multi-node/rdm_pingpong \
 	performance/multi-node/rdm_one_sided \
-################################################################################
-# Single-node bins
-################################################################################
 	performance/single-node/lock_tests
 
 # dist_bin_SCRIPTS = \
 #	scripts/run_craytests.sh
 
 noinst_LTLIBRARIES = libcraytests.la
 libcraytests_la_SOURCES = \
-	common/ct_utils.c
+	common/ct_utils.c \
+	common/print_utils.c
```

Comment:
Given the generic-ness of this name, I'd prefer the files be renamed to ct_print_utils.c and ct_print_utils.h.